e2e-tests/cypress/tests/plugins/db_request.js

```diff
@@ -127,9 +127,36 @@ function toLowerCase(config, name) {
     return name.toLowerCase();
 }
 
+const dbRefreshPostStats = async ({dbConfig}) => {
+    if (!knexClient) {
+        knexClient = getKnexClient(dbConfig);
+    }
+
+    // Only run for PostgreSQL
+    if (dbConfig.client !== 'postgres') {
+        return {
+            skipped: true,
+            message: 'Refresh post stats is only supported for PostgreSQL',
+        };
+    }
+
+    try {
+        await knexClient.raw('REFRESH MATERIALIZED VIEW posts_by_team_day;');
+        await knexClient.raw('REFRESH MATERIALIZED VIEW bot_posts_by_team_day;');
```

Comment:
I wish there's a way to speed up the jobs (customizable daily scheduler) so it can be triggered faster for E2E testing purpose. Direct DB access from Cypress (or Playwright) is fine but we cannot use this for remote test servers where we don't have DB credentials.

Reply:
Totally agree. There isn't any exposed API yet to trigger scheduled jobs. Something to look into.